events/mark-listener.js

```diff
@@ -0,0 +1,112 @@
+// mark-listener.js
+const { Events } = require('discord.js');
+
+module.exports = {
+    name: 'messageCreate',
+    execute(message, client) {
+        const users = {
```

Comment:
Consider adding this long var into assets folder, although "assets" is not exactly the name for this stuff, and could also be renamed to something like "data files" or of the like